index.html

```diff
@@ -7,5 +7,7 @@
 </head>
 <body>
     <h1>Test repo</h1>
+    <p>new stuff to push to the main branch</p>
+    <p>for merging up there (not locally here)</p>
```

Comment:
Needs another paragraph to describe other aspects of the test
```suggestion
    <p>Reviewing remotely</p>
```